clouddriver-titus/src/main/groovy/com/netflix/spinnaker/clouddriver/titus/caching/providers/TitusInstanceProvider.groovy

```diff
@@ -92,14 +122,33 @@ class TitusInstanceProvider implements InstanceProvider<TitusInstance> {
       healthKeys.unique().each { key ->
         def externalHealth = cacheView.getAll(HEALTH.ns, key)
         if (externalHealth) {
-          def health = externalHealth*.attributes
-          health.each { it.remove('lastUpdatedTimestamp') }
+          List<Map<String, Object>> health = externalHealth*.attributes
+          health.each {
+            it.remove('lastUpdatedTimestamp')
+            // groovy + java means that everything must be explicitly a string
+            it.allowMultipleEurekaPerAccount = it.allowMultipleEurekaPerAccount.toString()
```

Comment:
I'm not sure if changing this value to an explicit string here (instead of a boolean) will affect the UI. I am able to run tasks locally with Orca + Clouddriver (create/clone/distroy).